cocotb/scheduler.py

```diff
@@ -917,9 +917,27 @@ def finish_test(self, exc):
         return self._finish_test(exc)
 
     def _finish_test(self, exc):
-        self._test.abort(exc)
+        self._abort_test(exc)
```

Comment:
Since removing the system tasks there is no way to finish a test from `_sim_event`, which is the only caller here (and the only caller of this function that is valid). The line wasn't previously covered. I'm going to excuse it, I don't think it shuld be pragma'd out.